From 9728fb3f305b35d09b2c8e6382a6bfb5d33dd5d5 Mon Sep 17 00:00:00 2001
From: Tamas Ban <tamas.ban@arm.com>
Date: Mon, 22 Apr 2024 15:41:45 +0200
Subject: [PATCH 854/954] refactor(changelog): change all occurrences of RSS to
 RSE

Changes all occurrences of "RSS" and "rss" in the code and build files
to "RSE" and "rse".

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 71908bbe563263d6578295afb2bda159adf891aa refactor(changelog): change all occurrences of RSS to RSE]
Signed-off-by: Tamas Ban <tamas.ban@arm.com>
Change-Id: I66a6c87c158b14e18cba25ea6c6e0809e30008ca
(cherry picked from commit ccc717391f5359fe2f04fe5a7a6a841bbab587b6)
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/443846
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
Reviewed-by: Maxime MERE <maxime.mere@foss.st.com>
Tested-by: Yann GAUTIER <yann.gautier@foss.st.com>
---
 changelog.yaml | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/changelog.yaml b/changelog.yaml
index 0e9fc66719..ca5c259a97 100644
--- a/changelog.yaml
+++ b/changelog.yaml
@@ -941,11 +941,12 @@ subsections:
             deprecated:
               - drivers/arm/mhu
 
-          - title: RSS
-            scope: rss
+          - title: RSE
+            scope: rse
 
             deprecated:
               - drivers/arm/rss
+              - rss
 
           - title: TZC
             scope: tzc
-- 
2.53.0

